modules/phase_field/src/kernels/KKSACBulkC.C

```diff
@@ -44,14 +52,7 @@ KKSACBulkC::computeDFDOP(PFFunctionType type)
 
     case Jacobian:
     {
```

Comment:
If you are not declaring local variables the curly braces (which define a scope) are not necessary and we omit them.